src/components/Form/Select.tsx

```diff
@@ -142,7 +148,10 @@ export const Select = forwardRef<HTMLDivElement, SelectProps>(
             </div>
           ))}
         </Popover>
-      </>
+        {stateText && (
+          <span className={twJoin("bbn-select-state-text", `bbn-select-state-text-${state}`)}>{stateText}</span>
+        )}
+      </div>
```

Comment:
Okay, got it. Lets remove `stateText` or `hint` from here. We'll create a separate component to wrap any form element. It'll add a label, hint and error